sql3/planner/inbuiltfunctionsdate.go

```diff
@@ -150,6 +166,34 @@ func (n *callPlanExpression) EvaluateDatepart(currentRow []interface{}) (interfa
 
 }
 
+// EvaluateDateTimeFromParts evaluates the call to date_time_from_parts. This uses the base time.Date() function.
+func (n *callPlanExpression) EvaluateDateTimeFromParts(currentRow []interface{}) (interface{}, error) {
+	timestamps := make([]int, len(n.args))
+	for i, arg := range n.args {
+		param, err := arg.Evaluate(currentRow)
+		if err != nil {
+			return nil, err
+		} else if param == nil {
+			return nil, nil
+		}
+		coercedValue, err := coerceValue(arg.Type(), parser.NewDataTypeInt(), param, parser.Pos{Line: 0, Column: 0})
+		if err != nil {
+			return nil, err
+		}
+		val, ok := coercedValue.(int64)
+		if !ok {
+			return nil, sql3.NewErrInternalf("unable to convert value")
+		}
+		timestamps[i] = int(val)
+	}
+
+	dt := time.Date(timestamps[0], time.Month(timestamps[1]), timestamps[2], timestamps[3], timestamps[4], timestamps[5], timestamps[6]*1000*1000, time.UTC)
+	if dt.Year() < 0 || dt.Year() > 9999 {
```

Comment:
Silent normalization of out of range values for Month and below should be highlighted in the documentation. May be an example showing October 32 would convert to November 1 would nicely illustrate this.

Reply:
Yes. I'll make sure to note this.